Tensile/AsmCaps.py

```diff
@@ -22,884 +22,909 @@
-              'HasMFMA_bf16_1k': False,
-              'HasMFMA_bf16_original': False,
-              'HasMFMA_constSrc': False,
-              'HasMFMA_f64': False,
-              'HasMFMA_f8': False,
-              'HasMFMA_i8_908': False,
-              'HasMFMA_i8_940': False,
-              'HasMFMA_vgpr': False,
-              'HasMFMA_xf32': False,
-              'HasSMulHi': True,
-              'HasWMMA': False,
-              'KernargPreloading': False,
-              'MaxLgkmcnt': 15,
-              'MaxVmcnt': 63,
-              'SupportedISA': True,
-              'SupportedSource': True,
-              'VOP3v_dot4_i32_i8': False,
-              'v_dot2_f32_f16': True,
-              'v_dot2c_f32_f16': False,
-              'v_dot4_i32_i8': False,
-              'v_dot4c_i32_i8': False,
-              'v_fma_f16': True,
-              'v_fma_f32': True,
-              'v_fma_f64': True,
-              'v_fma_mix_f32': True,
-              'v_fmac_f16': False,
-              'v_fmac_f32': True,
-              'v_mac_f16': False,
-              'v_mac_f32': False,
-              'v_mad_mix_f32': False,
-              'v_mov_b64': False,
-              'v_pk_fma_f16': True,
-              'v_pk_fmac_f16': False},
- (12, 0, 1): {'HasAddLshl': True,
-              'HasAtomicAdd': False,
-              'HasDirectToLdsDest': False,
-              'HasDirectToLdsNoDest': False,
-              'HasExplicitCO': True,
-              'HasExplicitNC': True,
-              'HasGLCModifier': False,
-              'HasNTModifier': False,
-              'HasLshlOr': True,
-              'HasMFMA': False,
-              'HasMFMA_b8': False,
-              'HasMFMA_bf16_1k': False,
-              'HasMFMA_bf16_original': False,
-              'HasMFMA_constSrc': False,
-              'HasMFMA_f64': False,
-              'HasMFMA_f8': False,
-              'HasMFMA_i8_908': False,
-              'HasMFMA_i8_940': False,
-              'HasMFMA_vgpr': False,
-              'HasMFMA_xf32': False,
-              'HasSMulHi': True,
-              'HasWMMA': False,
-              'KernargPreloading': False,
-              'MaxLgkmcnt': 15,
-              'MaxVmcnt': 63,
-              'SupportedISA': True,
-              'SupportedSource': True,
-              'VOP3v_dot4_i32_i8': False,
-              'v_dot2_f32_f16': True,
-              'v_dot2c_f32_f16': False,
-              'v_dot4_i32_i8': False,
-              'v_dot4c_i32_i8': False,
-              'v_fma_f16': True,
-              'v_fma_f32': True,
-              'v_fma_f64': True,
-              'v_fma_mix_f32': True,
-              'v_fmac_f16': False,
-              'v_fmac_f32': True,
-              'v_mac_f16': False,
-              'v_mac_f32': False,
-              'v_mad_mix_f32': False,
-              'v_mov_b64': False,
-              'v_pk_fma_f16': True,
-              'v_pk_fmac_f16': False}}
+def getCapabilitiesCache(compiler_version: tuple) -> dict:
```

Comment:
```suggestion
def getCapabilitiesCache(compilerVersion: tuple) -> dict:
```
Staying consistent with camelCase is probably best.

Reply:
I was trying to avoid a cycle by pulling in Common. Will update to camel.